.github/workflows/enos-ci-cleanup.yml

```diff
@@ -0,0 +1,92 @@
+name: enos-ci-cleanup
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '05 02 * * *'
+  push:
```

Comment:
Don't forget to remote this.

Reply:
I'll do this and `--no-dry-run` as the last two commits